kubernetes/nginx-flex-dysk-newdisk.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: nginx-flex-dysk
+spec:
+  containers:
+  - name: nginx-flex-dysk
+    image: nginx
+    volumeMounts:
+    - name: test
+      mountPath: /data
+  volumes:
+  - name: test
+    flexVolume:
+      driver: "dysk/dysk"
+      fsType: "ext4"
+      readOnly: false
+      secretRef:
+        name: dyskcreds
+      options:
+        accountName: "account-name"
+        size: "5"
```

Comment:
Size is already a part of PV spec check https://kubernetes.io/docs/concepts/storage/persistent-volumes/
example: 
```
apiVersion: v1
kind: PersistentVolume
metadata:
  name: pv0003
spec:
  capacity:
    storage: 5Gi
  volumeMode: Filesystem
  accessModes:
    - ReadWriteOnce
  persistentVolumeReclaimPolicy: Recycle
  storageClassName: slow
  mountOptions:
    - hard
    - nfsvers=4.1
  nfs:
    path: /tmp
    server: 172.17.0.2
```